src/fons.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: Roboto;
+  src: url('./fons/Roboto-Medium.ttf') format('ttf');
```

Comment:
The format specified in the `src` attribute should be `'truetype'` instead of `'ttf'`. The correct line should be: `src: url('./fons/Roboto-Medium.ttf') format('truetype');`.